core/modules/filter/filter.pages.inc

```diff
@@ -432,22 +432,54 @@ function filter_format_editor_link_form($form, &$form_state, $format) {
     '#parents' => array('attributes', 'text'),
   );
 
-  $form['target'] = array(
+  $form['advanced'] = array(
+    '#title' => t('Advanced options'),
```

Comment:
I really don't like calling them 'Advanced'. They are just more options. Perhaps less-common, but not advanced.